<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSDlgArmManualWhiteBalance</class>
 <widget class="QDialog" name="CSDlgArmManualWhiteBalance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>370</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ARM Manual White Balance</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="2">
      <widget class="QSlider" name="horizontalSlider_B">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_GR">
       <property name="text">
        <string>GR</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_R">
       <property name="text">
        <string>R</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSlider" name="horizontalSlider_GR">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSlider" name="horizontalSlider_R">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSlider" name="horizontalSlider_GB">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_GB">
       <property name="text">
        <string>GB</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_B">
       <property name="text">
        <string>B</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QSpinBox" name="spinBox_R"/>
     </item>
     <item row="1" column="3">
      <widget class="QSpinBox" name="spinBox_GR"/>
     </item>
     <item row="2" column="3">
      <widget class="QSpinBox" name="spinBox_GB"/>
     </item>
     <item row="3" column="3">
      <widget class="QSpinBox" name="spinBox_B"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_close">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
